README.md

```diff
@@ -1,2 +1,50 @@
-# Awesome-BookShelf-ES6
-In this project, you will restructure your Awesome books app code. The goal is to make it more organized by using modules. You will also practice the ES6 syntax that you have learned.
+# Awesome-BookShelf-ES6 // Awesome books: with ES6
+
```

Comment:
- [OPTIONAL] I think one project title is enough, please kindly delete one of them, as this will make your readme file look more professional.